static/src/javascripts/projects/common/modules/commercial/contributions-service.js

```diff
@@ -329,26 +336,34 @@ export const fetchAndRenderEpic = (id: string) => {
         .then(json => {
             if (json && json.data) {
                 const { html, css, js, meta } = json.data;
-                const trackingCampaignId = `${meta.campaignId}`;
+                const {
+                    abTestName,
+                    abTestVariant,
+                    componentType,
+                    products = [],
+                    campaignCode,
+                    campaignId
+                } = meta;
+                const trackingCampaignId = `${campaignId}`;
```

Comment:
Super minor but do we need this assignment? (Could we use `campaignId` directly?) If the name is important could we have that mapping in the destructuring? e.g. `const { campaignId: trackingCampaignId } = meta;`?

Reply:
It's a fair point, it seems unnecessary to me also, it was like this when I started editing the file and I haven't changed it in this PR.

Reply:
I've changed this as the naming wasn't important and the assignment was unnecessary.